src/v3/src/transformer/oktaVerify/transformAppleSsoExtension.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { IDX_STEP } from '../../constants';
+import {
+  AutoSubmitElement,
+  IdxStepTransformer,
+  RedirectElement,
+  SpinnerElement,
+  TitleElement,
+} from '../../types';
+import { loc } from '../../util';
+
+export const transformAppleSsoExtension: IdxStepTransformer = ({ formBag, transaction }) => {
+  const { uischema } = formBag;
+  const { nextStep: { name: stepName } = {}, neededToProceed } = transaction;
+
+  if (typeof stepName === 'undefined') {
+    return formBag;
+  }
```

Comment:
Did you see this as a possibility when this transformer gets called?

Reply:
i thought to be safe and add this check since sometimes `nextStep` is missing